configs/default/CUST-SIRMIXALOT.config

```diff
@@ -0,0 +1,187 @@
+# 
+
+# diff all bare
+
+# version
+# Betaflight / STM32F7X2 (S7X2) 4.1.0 Sep 25 2019 / 14:09:57 (08e8afa09) MSP API: 1.42
```

Comment:
Have you tested this?

The banner starting in `# Betaflight` has to be the first line in the file, or this will not work.

Reply:
No; sorry, was literally following the instructions and didn't remove the first lines.